common/app/contentapi/ContentApiClient.scala

```diff
@@ -184,7 +183,7 @@ class ContentApiClient(httpClient: HttpClient)(implicit executionContext: Execut
 class PreviewContentApi(httpClient: HttpClient)(implicit executionContext: ExecutionContext) extends ContentApiClient(httpClient) {
   override val thriftClient = CircuitBreakingContentApiClient(
     httpClient = httpClient,
-    targetUrl = Configuration.contentApi.previewHost,
+    targetUrl = Configuration.contentApi.previewHost.getOrElse(Configuration.contentApi.contentApiHost),
```

Comment:
Is it safe to fall back to the live API?

Reply:
Really the previewHost should always be set for the apps which use this class, but because all apps share the configuration class there isn't really a way to do this conditionally.
It's consistent with the previous behaviour, so I think as long as we keep it configured correctly it'll be safe